google/cloud/spanner_v1/pool.py

```diff
@@ -229,12 +260,24 @@ def get(self, timeout=None):
         if timeout is None:
             timeout = self.default_timeout
 
+        start_time = time.time()
+        current_span = get_current_span()
+        span_event_attributes = {"kind": type(self).__name__}
+        current_span.add_event("Acquiring session", span_event_attributes)
         session = self._sessions.get(block=True, timeout=timeout)
+        span_event_attributes["session.id"] = session.session_id
 
         if not session.exists():
+            current_span.add_event(
+                "Session is not valid, recreating it", span_event_attributes
+            )
             session = self._database.session()
             session.create()
+            span_event_attributes["id"] = self._session_id
```

Comment:
Here _session_id is not a propery of pool.
Test cases are already failing for this
![image](https://github.com/user-attachments/assets/a1185642-22a6-4f34-a089-77fe7786cf76)

Reply:
Looks like this should be session.session_id